src/content/NewsContent.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2021 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { PostDescription } from '../theme/types';
+
+const news: PostDescription[] = [
+  {
+    title: 'August 2021 Newsletter',
+    text: 'Learn about the new Process Analytics website and logo, the new R library and improvements in the BPMN Visualization Typescript library.',
+    cover: 'https://miro.medium.com/max/350/0*Yymlf2vjfAIINqc2',
+    url: 'https://medium.com/@process-analytics/process-analytics-august-2021-newsletter-b793a647e35d',
```

Comment:
⚠️ this image is currently too large: 191kb
for posts, we have images from 16kb to 113kb. 

See also #30 and #131

Reply:
I reduced to 300. I I reduce more, the size rendered is smaller 🙁

Reply:
137kb now. I guess we should manage the image resize directly on the medium article.